spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -1066,7 +1094,6 @@ object QueryPlanSerde extends Logging with CometExprShim {
       case _: CollectLimitExec => true
       case _: UnionExec => true
       case _: TakeOrderedAndProjectExec => true
-      case _: WindowExec => true
```

Comment:
Removing this line fixes https://github.com/apache/datafusion-comet/issues/2737. I wish I could explain why, but I cannot.